404.html

```diff
@@ -0,0 +1 @@
+Sorry this page does not exist =(
```

Comment:
Even though we use branch name gh-pages, actual deployment goes into https://svn.apache.org/repos/asf/incubator/zeppelin. So i think it is irrelevant to github-pages.